packages/terra-status-view/docs/README.md

```diff
@@ -12,19 +12,32 @@ The status view component provides a customizable icon, message, and buttons in
 
 ```jsx
 import React from 'react';
-import Button from 'terra-button';
 import StatusView from 'terra-status-view';
 
+const buttons = [
+  {
+    id: 'ok',
+    key: 1,
+    text: 'OK',
+    size: 'medium',
+    onClick: handleOnAction1,
+  }, {
+    id: 'cancel',
+    key: 2,
+    text: 'Cancel,
+    size: 'medium',
+    onClick: handeOnAction2,
+  },
+];
+
 <StatusView
   isAlignedTop={false}
   isGlyphHidden={false}
   variant="error"
   title="A 500 error was thrown from the service."
   message=message="A null pointer exception was thrown."
->
-  <Button text="OK" key="1 />
-  <Button text="Cancel" key="2" />
-</StatusView>
+  buttonAttrs={buttons}
+/>
```

Comment:
I think many components have been removing this usage sections. Example code can be found within the doc template examples.

Reply:
updated